src/host/stream.cpp

```diff
@@ -521,140 +407,51 @@ til::CoordType RetrieveNumberOfSpaces(_In_ til::CoordType sOriginalCursorPositio
 // populated.
 // - STATUS_SUCCESS on success
 // - Other NTSTATUS codes as necessary
-[[nodiscard]] static NTSTATUS _ReadCharacterInput(InputBuffer& inputBuffer,
-                                                  std::span<char> buffer,
-                                                  size_t& bytesRead,
-                                                  INPUT_READ_HANDLE_DATA& readHandleState,
-                                                  const bool unicode,
-                                                  std::unique_ptr<IWaitRoutine>& waiter)
+[[nodiscard]] NTSTATUS ReadCharacterInput(InputBuffer& inputBuffer,

```

Comment:
should this be a member on InputBuffer, or stay relegated to stream.cpp?

Reply:
I'd like this to stay here to minimize the changes a little bit. There are many further changes required anyways to fix our text handling.